releasenotes/notes/43821.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 43807
+releaseNotes:
+  - |
+    **Added** the config type for each configuration summary when using `istioctl pc all`.
+  - |
+    **Added** the endpoint configuration summary to `istioctl pc all`.
```

Comment:
```suggestion
    **Added** endpoint configuration summaries to `istioctl proxy-config all`.
```

Reply:
Is it beneficial to have these listed as different items?  Would it make as much sense to say something like this as a single entry?
```
**Added** config type and endpoint configuration summaries to `istioctl proxy-config all`
```

Reply:
Thanks for the suggestion, I think this single entry looks better